pyanaconda/packaging/dnfpayload.py

```diff
@@ -734,6 +735,7 @@ def install(self):
             msg = 'Failed to download the following packages: %s' % str(e)
             exc = packaging.PayloadInstallError(msg)
             if errors.errorHandler.cb(exc) == errors.ERROR_RAISE:
+                log.error("Installation failed: %r", exc)
                 _failure_limbo()
```

Comment:
> Wouldn't it be better to make _failure_limbo() take an (optional) exception and log it?

I don't don't think that's really needed, and it's clearer to have the log.error right at the point of the exception.